### ISSUE TITLE
Question about 'Customer' in Project sprint slides

### ISSUE THREAD

**Issue by pptnz** [swsnu/swppfall2017#52]
10/10 강의 슬라이드인 Project Sprints의 Page 12에서 Customer에 대해 잘 이해되지 않는 부분이 있어 질문 올립니다.
"both in general and specifically"에서 "general"은 이해가 되는데 "specifically (the customer who informed this document)"가 이해되지 않습니다. Requirement 문서를 읽는 사람을 의미하는 것입니까?

**Comment by bgchun** [comment]
Yes. In our project setup, the customer who informed this document is your TA. :)

**Comment by jim6829** [comment]
답변 감사합니다! 그러면 하나만 더 여쭙고 싶은데요. A brief description of the customer specifically에 포함되어야 하는 내용으로는 무엇이 있을까요? 그냥 조교님을 한 명의 특정한 고객으로서 이 서비스를 왜 이용할 것인지에 대한 내용이면 충분할까요?

**Comment by bgchun** [comment]
Don't worry about the specific customer part. 
You can describe your requirements for general potential clients.
